lg_mirror/scripts/capture_webcam_node.py

```diff
@@ -19,6 +19,7 @@ def main():
     framerate = int(rospy.get_param('~framerate'))
     max_quantizer = int(rospy.get_param('~max_quantizer', 60))
     target_bitrate = int(rospy.get_param('~target_bitrate', 768000))
+    flip = bool(rospy.get_param('~flip', False))
```

Comment:
Coercing this into a bool is either unnecessary (because `rospy.get_param()` converts to `bool` for you, which I don't think it does?) or broken (because any non-empty string will evaluate to `True`).  See http://stackoverflow.com/questions/715417/converting-from-a-string-to-boolean-in-python